lib/config/yml_generator_config.dart

```diff
@@ -100,8 +100,9 @@ class YmlGeneratorConfig {
           property.containsKey('ignore') && property['ignore'] == true;
       final nonFinal = ignored ||
           property.containsKey('non_final') && property['non_final'] == true;
-      final includeIfNull = property.containsKey('include_if_null') &&
-          property['include_if_null'] == true;
+      final includeIfNull = !property.containsKey('include_if_null') ||
```

Comment:
This looks too verbose, can't we just use `property['include_if_null'] == true` ?

Reply:
the default is true. so i switched to `property['include_if_null'] != false`